lib/stateid.mli

```diff
@@ -42,3 +42,10 @@ type ('a,'b) request = {
   name     : string
 }
 
+(* Asks the document manager is the given state is valid (or belongs to an
```

Comment:
```suggestion
(* Asks the document manager if the given state is valid (or belongs to an
```

Reply:
Isn't it your change in your PR?

Reply:
Yes, read the message below, or ignore this comment.